internal/gatewayapi/testdata/gateway-with-listener-with-udproute-with-mismatch-port-protocol.out.yaml

```diff
@@ -72,8 +72,7 @@ udpRoutes:
     parents:
     - conditions:
       - lastTransitionTime: null
-        message: 'backend Service validation failed: UDP Port 8080 not found on Service
-          default/service-1'
+        message: UDP Port 8080 not found on Service default/service-1
```

Comment:
The message of UDP/TCP/TLS will be handled in a follow-up PR to keep the same format of the HTTPRoute status.